routing/routing_session.hpp

```diff
@@ -105,6 +106,15 @@ class RoutingSession
   inline void SetState(State state) { m_state = state; }
 
   Route const & GetRoute() const { return m_route; }
+  /// \returns true if any altitude information along |m_route| is available and
+  /// false otherwise.
+  bool HasRouteAltitude() const;
+  /// \brief copies route altitude information to |routeAltitudes| if any is available and
+  /// returns true. If no route altitude information is available returns false.
```

Comment:
At run time no junction has invalid altitude. If the point is not covered by srtm we put default value for altitude at run time. The return value of this method has another meaning. This method return false if there's no navigation route as the time the method is called.